Source/WebCore/rendering/GridTrackSizingAlgorithm.cpp

```diff
@@ -1724,8 +1724,10 @@ void GridTrackSizingAlgorithm::run()
 
     // Step 3.
     m_strategy->maximizeTracks(tracks(m_direction), m_direction == GridTrackSizingDirection::ForColumns ? m_freeSpaceColumns : m_freeSpaceRows);
-    if (m_strategy->isComputingSizeOrInlineSizeContainment() && !m_renderGrid->explicitIntrinsicInnerLogicalSize(m_direction))
+    if (m_strategy->isComputingSizeOrInlineSizeContainment() && !m_renderGrid->explicitIntrinsicInnerLogicalSize(m_direction)) {
```

Comment:
I don't think we need to add to the early return. Let's refactor it.

I would really prefer something like


```
auto noContent = m_strategy->isComputingSizeOrInlineSizeContainment() && !m_renderGrid->explicitIntrinsicInnerLogicalSize(m_direction)

if (!noContent)
   stretchFlexibleTracks(initialFreeSpace);

stretchAutoTracks();
```

Reply:
Yeah I think that's a nice little cleanup and doesn't really hinder readability. Done